### ISSUE TITLE
Facebook isn't doing DP problems much any more

### ISSUE THREAD

**Issue by iandouglas** [seanprashad/leetcode-patterns#117]
Hey @seanprashad just as a heads-up: I know your lists are curated from other sources, but might be worth looking into new sources as well. I've heard from people going through FB interviews currently that they've been told by recruiters that they will no longer be asking DP problems, and I've had this confirmed by other people who do interviewing at Facebook that they're seeing less "signal" from people around DP problems. Currently, DP problems are the second-most common tech challenge type listed from these combined sources on your site.

**Comment by seanprashad** [comment]
Hey @iandouglas! Thanks a ton for bringing this to my attention - I've also heard from FB interviewers that they are banning/removing DP questions from their internal question bank (_and other monstrous questions like Alien Dictionary_).

With regards to new sources, I'm definitely all ears! My original thought when creating this project was to limit the list to questions that forced individuals to use the `Sliding WIndow` pattern, or `DFS` etc. During my own studies, I absolutely hated questions that could be solved via `DFS`/`BFS`/`Pattern X` etc but were so obscure in their phrasing that I didn't even know where to start. In retrospect, I'd vouch for a shorter list that individuals can use to teach themselves `Pattern X` and apply that to many questions. I don't think this website is the best for individuals targetting a specific company - if you're familiar with patterns, then consider investing in Leetcode Premium to get an up-to-date list (_it was well worth it for my interviewing prep_).

The tagged companies for any specific question on Leetcode change weekly - this means that a "_hot_" question asked by `Company X` might not be "hot" in a few weeks, or even months. Once every 3-4 months, I spend ~2 hours going through the questions and updating them [here](https://github.com/seanprashad/leetcode-patterns/blob/master/src/data/index.js) - it isn't pretty but investing those few hours means others can potentially benefit tenfold in their own journey! You can see when I last update the question list by hovering over the `?` icon beside the `Companies` header:

![image](https://user-images.githubusercontent.com/13009507/132438982-0dc08deb-3658-41a8-a887-06d717c877c5.png)

So long story short, I (1) need to update the list sometime this month (as it's due) and (2) think about how #113 conveys new information, like how you noticed a trend of FB having a bunch of DP-tagged questions, but in reality, they are asking them on a less frequent basis.

I'd love to hear your thoughts on this as it's starting to get me thinking about how we can improve the experience!

**Comment by seanprashad** [comment]
Marking this as solved due to not being actionable
